curvefs/src/client/s3/client_s3_adaptor.cpp

```diff
@@ -20,19 +20,21 @@
  * Author: huyao
  */
 
-#include "curvefs/src/client/s3/client_s3_adaptor.h"
+
 
 #include <brpc/channel.h>
 #include <brpc/controller.h>
-
 #include <algorithm>
+#include <list>
+#include "curvefs/src/client/s3/client_s3_adaptor.h"
+#include "curvefs/src/common/s3util.h"
 
 namespace curvefs {
 
 namespace client {
 
 CURVEFS_ERROR
-S3ClientAdaptorImpl::Init(const S3ClientAdaptorOption& option, S3Client* client,
+S3ClientAdaptorImpl::Init(const S3ClientAdaptorOption &option, S3Client *client,
```

Comment:
Is it necessary to change the format？ almost nowhere to find actually modify place

Reply:
Yes, the file is automatically formatted during replacement. Pay attention to this problem next time.